required/amsmath/amsmath.dtx

```diff
@@ -1039,6 +1039,13 @@ Foreign command \@backslashchar#1;\MessageBreak
 }
 \fi
 %    \end{macrocode}
+%   \changes{v2.17s}{2024/10/30}{guard against primitive if (gh/1448)}
+%    \begin{macrocode}
+{\uccode`7=`\\ \uccode`(=`i \uccode`)=`f
+  \uppercase{\gdef\testif@#1#2#3#4\testif@{%
+    \ifx7#1\ifx(#2\ifx)#3\@tempswafalse\fi\fi\fi}
+}}
```

Comment:
maybe a comment what this does and why with the uccodes? I means, I know but ...

Reply:
It's as comprehensively documented as the existing blocks just above such as 


```
{\uccode`7=`\\ \uccode`8=`m \uccode`9=`a \uccode`0=`t \uccode`!=`h` 
```

I'll add a line before all of them saying something like special uccodes are used to generate macros with arguments delimited by catcode 12 letters as returned by `\meaning`

Reply:
true a lot in amsmath is not really documented ... but such a general line will help